strongbox-data-service/src/main/java/org/carlspring/strongbox/config/DataServiceConfig.java

```diff
@@ -17,24 +17,29 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.EnableAspectJAutoProxy;
 import org.springframework.context.annotation.Import;
-import org.springframework.context.annotation.Lazy;
 import org.springframework.core.io.ClassPathResource;
-import org.springframework.data.orient.commons.core.OrientTransactionManager;
-import org.springframework.data.orient.object.OrientObjectDatabaseFactory;
-import org.springframework.data.orient.object.OrientObjectTemplate;
+import org.springframework.orm.jpa.JpaTransactionManager;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.transaction.PlatformTransactionManager;
 import org.springframework.transaction.annotation.EnableTransactionManagement;
+import org.springframework.transaction.support.TransactionTemplate;
+
+import com.orientechnologies.orient.client.remote.OServerAdmin;
+import com.orientechnologies.orient.core.entity.OEntityManager;
 
 /**
  * Spring configuration for data service project.
  *
  * @author Alex Oreshkevich
  */
 @Configuration
-@EnableTransactionManagement
+@EnableTransactionManagement(proxyTargetClass = true, order = 100)
```

Comment:
Could you put appropriate comment for that constant value please? You have 99 another configurations?